src/Component/GeneralEntity/GeneralEntityRoot/GeneralEntityRoot.tsx

```diff
@@ -253,8 +253,29 @@ export function GeneralEntityRoot<T extends BaseEntity>({
     }
   }, [entityController, onEntitiesLoaded, entityType, pageCurrent, pageSize, sortField, sortOrder, isFiltered]);
 
+  const discardChanges = () => {
+    Modal.destroyAll();
+    fetchEntity(parseInt(id.toString(), 10));
+    notification.info({
+      message: t('GeneralEntityRoot.saveWarning', {
+        entity: TranslationUtil.getTranslationFromConfig(entityName, i18n)
+      })
+    });
+  };
+
+  const saveChanges = () => {
+    onSaveClick();
+    fetchEntity(parseInt(id.toString(), 10));
```

Comment:
this is a little tedious, but makes the liniting error go away. also it is a little more robust this way.
```suggestion
    if (!isNil(id)) {
      fetchEntity(parseInt(id.toString(), 10));
    }
```

Reply:
`isNil` needs to be imported from lodash